<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-10-13T10:25:51Z" name="Default suite" finished-at="2022-10-13T10:25:52Z" duration-ms="590">
    <groups>
    </groups>
    <test started-at="2022-10-13T10:25:51Z" name="Default test" finished-at="2022-10-13T10:25:52Z" duration-ms="590">
      <class name="Ashu2.TC0056">
        <test-method signature="tc002()[pri:0, instance:Ashu2.TC0056@50de186c]" started-at="2022-10-13T15:55:51Z" name="tc002" finished-at="2022-10-13T15:55:52Z" duration-ms="437" status="FAIL">
          <exception class="java.lang.IllegalStateException">
            <message>
              <![CDATA[The driver executable must exist: C:\chromedriver_win32\chromedriver.exe]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalStateException: The driver executable must exist: C:\chromedriver_win32\chromedriver.exe
	at org.openqa.selenium.internal.Require$FileStateChecker.isFile(Require.java:342)
	at org.openqa.selenium.remote.service.DriverService.checkExecutable(DriverService.java:147)
	at org.openqa.selenium.remote.service.DriverService.findExecutable(DriverService.java:142)
	at org.openqa.selenium.chrome.ChromeDriverService.access$000(ChromeDriverService.java:39)
	at org.openqa.selenium.chrome.ChromeDriverService$Builder.findDefaultExecutable(ChromeDriverService.java:233)
	at org.openqa.selenium.remote.service.DriverService$Builder.build(DriverService.java:437)
	at org.openqa.selenium.chrome.ChromeDriverService.createDefaultService(ChromeDriverService.java:128)
	at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:49)
	at Ashu.General.openapplication(General.java:15)
	at Ashu2.TC0056.tc002(TC0056.java:16)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalStateException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc002 -->
      </class> <!-- Ashu2.TC0056 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
